boards/DIM/Inc/App/App_DimWorld.h

```diff
@@ -49,3 +53,9 @@ struct RgbLedSequence *
     App_DimWorld_GetRgbLedSequence(const struct DimWorld *world);
 struct RotarySwitch *
     App_DimWorld_GetDriveModeSwitch(const struct DimWorld *world);
+
+struct BinarySwitch *App_DimWorld_GetStartSwitch(const struct DimWorld *world);
+struct BinarySwitch *
+    App_DimWorld_GetTractionControlSwitch(const struct DimWorld *world);
+struct BinarySwitch *
+    App_DimWorld_GetTorqueVectoringSwitch(const struct DimWorld *world);
```

Comment:
Now that the codebase is more mature now, I felt that the jdocs for `App_<SomeBoard>World_GetInterface` functions became pretty redundant because I couldn't provide any more context than the function nam. I will add jdocs for the other `App` and `Io` code though!

Reply:
I totally agree, the reason I think we should have jdocs here is not that I think they add particularly valuable context, but rather that we set the example that functions should *always* have a jdoc. This is so that people who might not necessarily have the context or experience to properly determine where they are/are not appropriate add them everywhere, rather then leaving them out in potentially important places. I think this is particularly important for student teams where people with a wide variety of skill levels could potentially work on that code.

Reply:
Fair enough put the comments back for all the `App_World.h`